liquibase/changes/feat_533.sql

```diff
@@ -0,0 +1,2 @@
+-- Adding created_at column to Feed table with default value of current timestamp for new records
+ALTER TABLE Feed ADD COLUMN created_at TIMESTAMPTZ NULL;
```

Comment:
We should make the created_at mandatory. I understand that the current rows don't contain the data. We can do this in multiple SQL statements: add a column with no restriction, update the table, and add the restriction. The _default_ value can be the minimum date.

Reply:
I agree that it should be mandatory. Are you proposing that we set a default value for the field, such as February 8th (prod start date)?

Reply:
✅ done with both